lib/network.go

```diff
@@ -2090,12 +2106,26 @@ func (msg *MsgDeSoHeader) EncodeHeaderVersion2(preSignature bool) ([]byte, error
 	// that MsgDeSoHeader version 2 does not need to be backwards compatible.
 	retBytes = append(retBytes, UintToBuf(msg.TstampSecs)...)
 
-	// Height
+	// Height: similar to the field above, this field can be encoded to take
+	// up the full 64 bits now that MsgDeSoHeader version 2 does not need to
```

Comment:
It doesn't always take up all 64 bits. UintToBuf is a variable length encoding.

Reply:
It can take up to 64 bits. The field is encoded as variable length, but it has 64 bits of resolution.